src/interface/sparse_pack.hpp

```diff
@@ -156,11 +156,21 @@ struct base_t {
 
 // An example variable name type that selects all variables available
 // on Mesh*Data
-struct any : public base_t<true> {
+struct any_withautoflux : public base_t<true> {
   template <class... Ts>
-  KOKKOS_INLINE_FUNCTION any(Ts &&...args) : base_t<true>(std::forward<Ts>(args)...) {}
+  KOKKOS_INLINE_FUNCTION any_withautoflux(Ts &&...args)
+      : base_t<true>(std::forward<Ts>(args)...) {}
   static std::string name() { return ".*"; }
 };
+
+struct any_nonautoflux : public base_t<true> {
+  template <class... Ts>
+  KOKKOS_INLINE_FUNCTION any_nonautoflux(Ts &&...args)
+      : base_t<true>(std::forward<Ts>(args)...) {}
+  static std::string name() { return "^(?!bnd_flux::).+"; }
+};
+
+using any = any_nonautoflux;
```

Comment:
May that have some unintended side-effects?

Reply:
I don't think so, since there are few places where `any` is currently used and those places assumed that fluxes would not get sucked up when using `any`. I think in the future we, should avoid using `any` and use either `any_nonautoflux` or `any_withautoflux`.